### ISSUE TITLE
how to use the custom loss?

### ISSUE THREAD

**Issue by liuxindazz** [pytorch/ignite#350]
In pytorch, I can define my loss like:
```
class LossFunc(nn.Module):
    def __init__(self):
        pass
    def forward(self,):
        pass
```
In ignite, I find that loss is using  F.lossfunction(torch.nn loss function). So I'm confused whether I could define a custom loss function, and if I could, how to do it?

**Comment by vfdev-5** [comment]
@liuxindazz there is no restriction on loss function that can be used in ignite. For example,
```
criterion = LossFunc()

trainer = create_supervised_trainer(model, optimizer, criterion, device=device)
```

**Comment by liuxindazz** [comment]
Thank you for the reply, I got a little bit mixed up just now.
It's clear that how to use it.
